### ISSUE TITLE
Please update documentation on "Connect G Suite to Cloud App Security"

### ISSUE THREAD

**Issue by bp88** [MicrosoftDocs/CloudAppSecurityDocs#330]
Please update documentation on how to Connect G Suite to Cloud App Security. The Google Admin console and the Google Cloud Platform console have changed considerably to the point where these instructions are incredibly hard to follow. I would strongly recommend re-creating the instructions from scratch to avoid any discrepancies between changes in the UI.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a1f41e1e-2541-8773-efc2-eeb83a403a15
* Version Independent ID: 2f2d0316-9e75-44d6-0ab6-a56b1352a1e2
* Content: [Connect G Suite to Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/connect-google-apps-to-microsoft-cloud-app-security?redirectedfrom=MSDN)
* Content Source: [CloudAppSecurityDocs/connect-google-apps-to-microsoft-cloud-app-security.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/master/CloudAppSecurityDocs/connect-google-apps-to-microsoft-cloud-app-security.md)
* Service: **cloud-app-security**
* Product: ****
* Technology: ****
* GitHub Login: @shsagir
* Microsoft Alias: **shsagir**

**Comment by John-moreno-a** [comment]
Documentation instructions still the same. Looking forward to this update.

**Comment by shsagir** [comment]
@bp88 Thanks so much for taking the time to leave us your comment on our documentation. We're researching your issue to update the page based on your suggestions. If relevant, changes will be made in the next possible publish.

We're closing this as an open issue for now, but feel free to continue to comment here as necessary. We appreciate your contribution to our community.

**Comment by joshhighet** [comment]
Documentation appears out-of-date again regarding the App Registration in GCP.

Step 7 onwards in the [connection docs](https://docs.microsoft.com/en-au/cloud-app-security/connect-google-workspace-to-microsoft-cloud-app-security)
